types/spectest/tests/consensusdata/proposer/versioned_block_spectest.go

```diff
@@ -56,14 +56,14 @@ func (test *ProposerSpecTest) Run(t *testing.T) {
 		// compare blk data
 		var blkSSZ []byte
 		switch vBlk.Version {
-		case spec.DataVersionBellatrix:
-			require.NotNil(t, vBlk.Bellatrix)
-			blkSSZ, err = vBlk.Bellatrix.MarshalSSZ()
-			require.NoError(t, err)
 		case spec.DataVersionCapella:
 			require.NotNil(t, vBlk.Capella)
 			blkSSZ, err = vBlk.Capella.MarshalSSZ()
 			require.NoError(t, err)
+		case spec.DataVersionDeneb:
```

Comment:
if you scroll up you see a comment for issue 219. IMO you can delete the comment and close the issue